File: TeamCode/src/main/java/robotuprising/ftc2021/opmodes/examples/exHwMap.java
package robotuprising.ftc2021.opmodes.examples;

import com.qualcomm.robotcore.hardware.DcMotor;
import com.qualcomm.robotcore.hardware.DistanceSensor;
import com.qualcomm.robotcore.hardware.HardwareMap;

public class exHwMap {
    public DcMotor intake;

    public exHwMap(HardwareMap hardwareMap) {
        intake = hardwareMap.get(DcMotor.class, "intake");
    }
}

File: TeamCode/src/main/java/robotuprising/ftc2021/opmodes/junk/examples/exHwMap.kt
package robotuprising.ftc2021.opmodes.junk.examples

import com.qualcomm.robotcore.hardware.HardwareMap
import com.qualcomm.robotcore.hardware.DcMotor

class exHwMap(hardwareMap: HardwareMap) {
    var intake: DcMotor

    init {
        intake = hardwareMap.get(DcMotor::class.java, "intake")
    }
}